FATN_to_USB/tests/test_copy_fatn.py

```
@@ -0,0 +1,464 @@
+
+# ============================================================================
+# USB Unmounting Tests
+# ============================================================================
+
+class TestUSBUnmounting:
+    """Tests for USB device unmounting."""
+
+    @pytest.mark.unit
+    @patch('subprocess.run')
+    @patch('time.sleep')
+    def test_unmount_usb_device_success(self, mock_sleep, mock_run, copy_fatn, capsys):
```

Ruff (0.14.14)

Unused method argument: `mock_sleep`

(ARG002)